lessonbuilder/tool/src/java/org/sakaiproject/lessonbuildertool/util/SimplePageItemUtilities.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) 2017, University of Dayton
+ *
+ * Licensed under the Educational Community License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *             http://opensource.org/licenses/ecl2
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.sakaiproject.lessonbuildertool.util;
+
+import org.jsoup.Jsoup;
+import org.sakaiproject.lessonbuildertool.SimplePageItem;
+
+/**
+ * Created by Leonardo Canessa [lcanessa1 (at) udayton (dot) edu]
+ */
+public class SimplePageItemUtilities {
+    /**
+     * Get's a name to display to the user from a SimplePageItem
+     * @param name, SimplePageItem
+     * @return String of the name
+     */
+    public static String getDisplayName(final SimplePageItem name) {
+        String itemName = name.getName();
+        if(itemName.isEmpty()) {
+            if(!name.getHtml().isEmpty()) {
+                itemName = Jsoup.parse(name.getHtml()).text();
```

Comment:
I'd have to use the static deprecated function: `org.sakaiproject.util.FormattedText#stripHtmlFromText(java.lang.String, boolean, boolean)` in order to remove the JSoup dependency.

JSoup is all ready being used in two places in Sakai. One in the [kernel-impl](https://github.com/sakaiproject/sakai/blob/476073dbf44548da46133cc61634cd3b1a5ea801/kernel/kernel-impl/pom.xml#L261) and one in [blackboard common import parser](https://github.com/sakaiproject/sakai/blob/8b1817a36a1da551527b41c09297ed3a7d7ca2b3/common/import-parsers/blackboard_6/impl/pom.xml#L36).

With some guidance, I would be happy to consolidate the use of JSoup across Sakai.